flake.nix

```diff
@@ -88,15 +88,20 @@
               compiler
               # TODO
               # Do we want to enable any `crossPlatforms` here?
-              ((infernoFor compiler pkgs).flake { })
+              ((infernoFor compiler pkgs {}).flake { })
+            ) ++
+          lib.lists.forEach [ defaultCompiler ]
+            (compiler: lib.attrsets.nameValuePair
+              (compiler + "-prof")
+              ((infernoFor compiler pkgs {profiling = true; ghcOptions = [ "-eventlog" ];}).flake { })
             )
```

Comment:
Do we only want profiling enabled on GHC 9.x? If that's the case we could drop the `forEach` and `nameValuePair` and merge the attribute sets directly

Reply:
there are some new GHC9 only profiling modes, e.g. info-tables stuff, so depends on what profiling we want to do. I was just being lazy here tho, because the ghc8x version doens't work on M1 anyway

Reply:
In that case I think it's OK to only have the profiling build for GHC 9, so we can remove the other `forEach`